vmdb/app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -433,9 +433,11 @@ def build_global_and_my_filters(type)
   end
 
   def schedule_set_record_vars(schedule)
-    schedule.name = params[:name]
-    schedule.description = params[:description]
-    schedule.enabled = params[:enabled]
+    if params[:action] == "schedule_edit"
+      schedule.name = params[:name]
+      schedule.description = params[:description]
+      schedule.enabled = params[:enabled]
+    end
```

Comment:
Do we have to add the conditional here?  Can we extract this to a method and just call it from the `schedule_edit` actions?